src/components/editor/inputs/InputLang.jsx

```diff
@@ -33,16 +33,16 @@ const InputLang = (props) => {
   const selectLanguage = (selected) => {
     if (selected.length === 1) {
       setLang(selected[0].id)
+      setRadioButtonValue("present")
     } else {
-      setLang(undefined)
+      setLang(null)
+      setRadioButtonValue("absent")
     }
```

Comment:
this method is called when a user picks a language in the type ahead compontent: 

1. we use `null` to indicate no language is selected, `lang` can never be undefined (see line 20 above)
2. set the currently selected radio button appropriately

Reply:
Maybe this could be a comment for the function??  it's pretty intricate.

Reply:
added inline function comments